hazelcast/src/main/java/com/hazelcast/internal/management/operation/ScriptExecutorOperation.java

```diff
@@ -79,31 +71,14 @@ public Object getResponse() {
     protected void writeInternal(ObjectDataOutput out) throws IOException {
         out.writeUTF(engineName);
         out.writeUTF(script);
-        if (bindings != null) {
-            out.writeInt(bindings.size());
-            Set<Map.Entry<String, Object>> entries = bindings.entrySet();
-            for (Map.Entry<String, Object> entry : entries) {
-                out.writeUTF(entry.getKey());
-                out.writeObject(entry.getValue());
-            }
-        } else {
-            out.writeInt(0);
-        }
+        // kept for compatibility
+        out.writeInt(0);
```

Comment:
Will be surrounded by a version check and conditional serialization in 3.10 with the forward port of this PR.